src/openeo_gfmap/preprocessing/meteo.py

```diff
@@ -0,0 +1,20 @@
+import openeo
+
+
+def compress_meteo_uint16(cube: openeo.DataCube) -> openeo.DataCube:
```

Comment:
does this need to be meteo specific or can we make a generic method for this? (e.g. cast_to_uint16)
If you want to include the scaling, perhaps make it an argument so we don't have a hardcoded factor. In that case the name should also represent the functionality (e.g. scale_and_cast_to_uint16)

Reply:
Ok, I renamed it top a more general function with linear scaling option.
`def compress_uint16(cube: openeo.DataCube, alpha: float = 1.0, beta: float = 0.0) -> openeo.DataCube:`